drivers/char/broadcom/vc_sm/vc_sm_defs.h

```diff
@@ -165,6 +181,41 @@ typedef struct {
 
 } VC_SM_FREE_ALL_T;
 
+/* Request to import memory (HOST->VC) */
+struct VC_SM_IMPORT {
```

Comment:
Linux doesn't use CAPS for structure names.

Reply:
I retained caps for consistency with the rest of the file. This file is also shared with the firmware.
So either we go through all the code on both sides and adopt the Linux scheme (and ignore the firmware scheme), or we retain the current style.

Then again I didn't retain full consistency as I dropped the typedefs in favour of named structs.

What's the preference here? A clean up set of patches to go the Linux way?

Reply:
I would choose to not introduce new rule violations rather than maintain consistency.

Reply:
OK, will do.
I'll do a follow up clean up set of patches to try and make vc-sm as a whole friendlier to mainline.